* Added two checks to see if the server respects the json type for Ob…

hylkevds opened the pull request:
…servation/result (for numbers and strings).

Sting result values should stay Strings, Numeric result values should stay numeric, since the server changing the value type would force the client to do "manual" re-parsing of the values.

taniakhalafbeigi commented:
This test is not part of conformance test suite as of Annex A in the specification of OGC SensorThings API.

taniakhalafbeigi commented:
We wait for this test to be added to spec and then merge it.

hylkevds commented:
I've added tests for the other reslut types (object, array, boolean) as well, and moved them to a separate class for easier maintenance.

Not sure if boolean, array and object result types need to be tested though, that might need some discussion first. But I had to write the tests any way, to test our implementation. :)
